update/application.go

```diff
@@ -0,0 +1,105 @@
+package update
+
+import (
+	"context"
+	"fmt"
+
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+	"github.com/ninech/nctl/internal/format"
+)
+
+// all fields need to be pointers so we can detect if they have been set by
+// the user.
+type applicationCmd struct {
```

Comment:
we are basically copying all tags form create/application.go. I'm just wondering if we can reuse it and somehow define them all in one place. Otherwise, it will be a bit difficult to maintain it.

The same for type `gitConfig`